utils/admin_console_plugin_index.js

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+function extractTextsFromPlugin(plugin) {
+    const texts = [];
+    if (plugin.name) {
+        texts.push(plugin.name);
+    }
+    if (plugin.id) {
+        texts.push(plugin.id);
+    }
+    if (plugin.settings_schema) {
+        if (plugin.settings_schema.footer) {
+            texts.push(plugin.settings_schema.footer);
+        }
+        if (plugin.settings_schema.header) {
+            texts.push(plugin.settings_schema.header);
+        }
+    }
+
+    if (plugin.settings_schema && plugin.settings_schema.settings) {
```

Comment:
can the `plugin.settings_schema.settings` check be moved into the if block on line 11? that way the parent check is in one spot?